presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/QueryCardinalityUtil.java

```diff
@@ -134,6 +134,7 @@ public CardinalityExtractorPlanVisitor(Lookup lookup)
             return Range.atLeast(0L);
         }
 
+        @Override
```

Comment:
why get rid of the override?

Reply:
i don't understand?

Reply:
wow i totally misread that. you added it.